e2e/integration_tests/data_flow_test.go

```diff
@@ -212,7 +212,7 @@ func Test_EndpointLinksAreAvailable(t *testing.T) {
 	failOnError(err)
 
 	if link_count != expected_link_count {
-		t.Fatalf("Database should only have made 36 links given the fake NPPES data that was loaded. Has: " + strconv.Itoa(link_count))
+		t.Fatalf("Database should only have made 4 links given the fake NPPES data that was loaded. Has: " + strconv.Itoa(link_count))
```

Comment:
Why 4 if expected_link_count is 30 ?

Reply:
This was a typo, thanks for catching this!